src/angular-model.js

```diff
@@ -191,15 +370,67 @@ angular.module('ur.model', []).provider('model', function() {
           this[prop] = original[prop];
         }
       },
+
+      /**
+       * @ngdoc function
+       * @name $exists
+       * @methodOf ur.model:$instance
+       * @description
+       * Reset the model to the state it was originally in when you first got it from the API
```

Comment:
Looks like copypasta from `$revert()`. `$exists()` checks whether an object exists in the API, based on whether it has an identity URL.